vuepress/docs/next/tutorials/compose/content-tutorial.md

```diff
@@ -1,108 +1,89 @@
 
--   Select the **Add Content** button. A drop-down list will let you
-    choose which Content Type the content will be based on.
+![image](./img/content-add.png)
 
-![image](./img/Content2.png)
+4. You will be asked to provide the following information.
+    * `Info` 
 
-You will be asked to provide the following information:
+         * `Content Type`: This is a pre-defined list of the `Types` available.
 
--   **Info**
+         * `Description`: The specific content's name or description. This field is required.
 
-    -   **Content Type**. This information is pre-defined, and lists the
-        content type for the content.
+    *   `Groups`
 
-    -   **Description**. The specific Content's name or description. This field
-        is mandatory.
+         * `Owner Group`: Choose the available user group to manage the content from the drop-down list. Click the `+` button to select. This field is required and predefined in the [Entando Identity Management System](../../docs/consume/identity-management.md).
 
--   **Groups**
+         * `View Only Groups`: Add groups to provide viewing access to the content.
 
-    -   **Owner Group.** Use the drop-down list to choose the
-        available user group that can manage the content.
-        This field is mandatory and may be predefined.
 
-    -   **Join Group.** Use the drop-down list to choose the
-        available user group to share the content with and select +. The group
-        selected will only be able to share and can not
-        administrate this content. This field is not mandatory.
+    *   `Categories`: Use `+` under `Join` to add to the applicable category tree for use in searching and sorting.  This field is not mandatory.
 
-    -   **Categories.** Select the category(s) and select + to add a classification to the 
-        content to use for searching and sorting.  This field is not mandatory.
+    *  `Content Attributes`: Define each required `Attributes` field and any optional fields for the `Content Type` of this item. Only the `Date` and `Title` are required.
 
-    -   **Content Attributes.** Define each required attributes field and any optional fields
-        defined by the content types used for this Content. This field is mandatory.
+    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version. 
 
-![image](./img/Content3.png)
+![Add Content Information](./img/content-addinfo.png)
 
-Save the new content by selecting:
+## Saving and Approving Content 
+1. For new or edited content, determine which `Status` it belongs to:  
+`Status`:
+ * `Draft`: The content is in the development stage and not ready for approval or publication. The `Save` or `Save and Continue` buttons establish this status for all content.
```

Comment:
for all content? aren't we just talking about a single `Content` item?

same as above: specific instances of content are singular and should be capitalized/backticked

would say "establish `Draft` status" -- otherwise it can read as those buttons establish `Save` or `Save and Continue` status